firefox-ios/Client/Frontend/UserContent/UserScripts/MainFrame/AtDocumentEnd/LegacyNightModeHelper.js

```diff
@@ -0,0 +1,120 @@
+/* vim: set ts=2 sts=2 sw=2 et tw=80: */
```

Comment:
```suggestion
```
Do you need this?

Reply:
I see you are mostly just moving code but thought why is this needed

Reply:
oh good catch. No we don't need it. Whoever uses vim can configure their editor locally.